ebean-datasource/src/main/java21/io/ebean/datasource/pool/ExecutorFactory.java

```diff
@@ -1,11 +1,61 @@
 package io.ebean.datasource.pool;
 
+import io.ebean.datasource.pool.ConnectionPool.Heartbeat;
```

Comment:
Cool: I didn't know this method until now:
Rolands note: Classes will be located in `META-INF/versions/21`
https://docs.oracle.com/javase/10/docs/specs/jar/jar.html#multi-release-jar-files